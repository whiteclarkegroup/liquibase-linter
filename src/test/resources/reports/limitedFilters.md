# `src/main/resources/ddl/first.xml`
| Change Set | Status | Rule       | Message           |
|------------|--------|------------|-------------------|
| *none*     | ERROR  | error-rule | Error message 1   |
| 2020010101 | ERROR  | error-rule | Error message 2   |
| 2020010102 | ERROR  | error-rule | Error message 3.1 |
|            |        | error-rule | Error message 3.2 |

# `src/main/resources/ddl/second.xml`
| Change Set | Status | Rule       | Message         |
|------------|--------|------------|-----------------|
| 2020010103 | ERROR  | error-rule | Error message 4<br>with newline |

# *Other*
| Change Set | Status | Rule       | Message       |
|------------|--------|------------|---------------|
| *none*     | ERROR  | error-rule | No change log |

# Summary
* ERROR: 6
* IGNORED: 3
* PASSED: 3
* DISABLED: 2
