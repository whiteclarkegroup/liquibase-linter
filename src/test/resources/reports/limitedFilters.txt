src/main/resources/ddl/first.xml
ERROR
	'error-rule': Error message 1
changeSet '2020010101'
ERROR
	'error-rule': Error message 2
changeSet '2020010102'
ERROR
	'error-rule': Error message 3.1
	'error-rule': Error message 3.2

src/main/resources/ddl/second.xml
changeSet '2020010103'
ERROR
	'error-rule': Error message 4
		with newline

Other
ERROR
	'error-rule': No change log

Summary:
	ERROR: 6
	IGNORED: 3
	PASSED: 3
	DISABLED: 2
