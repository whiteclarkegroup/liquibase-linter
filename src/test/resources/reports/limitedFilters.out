src/main/resources/ddl/first.xml
<span style="color: red;">ERROR</span>
	'error-rule': Error message 1
changeSet '2020010101'
<span style="color: red;">ERROR</span>
	'error-rule': Error message 2
changeSet '2020010102'
<span style="color: red;">ERROR</span>
	'error-rule': Error message 3.1
	'error-rule': Error message 3.2

src/main/resources/ddl/second.xml
changeSet '2020010103'
<span style="color: red;">ERROR</span>
	'error-rule': Error message 4
		with newline

Other
<span style="color: red;">ERROR</span>
	'error-rule': No change log

Summary:
	<span style="color: red;">ERROR</span>: 6
	<span style="color: yellow;">IGNORED</span>: 3
	<span style="color: green;">PASSED</span>: 3
	<span style="color: magenta;">DISABLED</span>: 2
